index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Tabzilla</title>
+    <link rel="stylesheet" type="text/css" href="css/tabzilla.css" media="screen" />
+    <!-- Do not do this in practice -->
```

Comment:
How about "Inline CSS should be avoided in practice, this is purely for example"?